Tests/PackageLoadingTests/PD_5_11_LoadingTests.swift

```diff
@@ -0,0 +1,88 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Basics
+import PackageModel
+import SourceControl
+import SPMTestSupport
+import XCTest
+
+class PackageDescription5_11LoadingTests: PackageDescriptionLoadingTests {
+    override var toolsVersion: ToolsVersion {
+        .v5_11
+    }
+
+    func testPackageContextGitStatus() throws {
+        let content = """
+                import PackageDescription
+                let package = Package(name: "\\(Context.gitInformation?.hasUncommittedChanges == true)")
+                """
+
+        try loadRootManifestWithBasicGitRepository(manifestContent: content) { manifest, observability in
+            XCTAssertNoDiagnostics(observability.diagnostics)
+            XCTAssertEqual(manifest.displayName, "true")
+        }
+    }
+
+    func testPackageContextGitTag() throws {
+        let content = """
+                import PackageDescription
+                let package = Package(name: "\\(Context.gitInformation?.currentTag ?? "")")
+                """
+
+        try loadRootManifestWithBasicGitRepository(manifestContent: content) { manifest, observability in
+            XCTAssertNoDiagnostics(observability.diagnostics)
+            XCTAssertEqual(manifest.displayName, "lunch")
+        }
+    }
+
+    func testPackageContextGitCommit() throws {
+        let content = """
+                import PackageDescription
+                let package = Package(name: "\\(Context.gitInformation?.currentCommit ?? "")")
+                """
+
+        try loadRootManifestWithBasicGitRepository(manifestContent: content) { manifest, observability in
+            XCTAssertNoDiagnostics(observability.diagnostics)
+
+            let repo = GitRepository(path: manifest.path.parentDirectory)
+            let currentRevision = try repo.getCurrentRevision()
+            XCTAssertEqual(manifest.displayName, currentRevision.identifier)
+        }
+    }
+
+    private func loadRootManifestWithBasicGitRepository(manifestContent: String, validator: (Manifest, TestingObservability) throws -> ()) throws {
```

Comment:
Formatting nit. Maybe worth running SwiftFormat on this file? Would also fix indentation for multi-line strings.
```suggestion
    private func loadRootManifestWithBasicGitRepository(
        manifestContent: String, 
        validator: (Manifest, TestingObservability) throws -> ()
    ) throws {
```